samples/sampler/stories/default/RangePicker.js

```diff
@@ -0,0 +1,30 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, select} from '@enact/storybook-utils/addons/knobs';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import RangePicker from '@enact/agate/RangePicker';
+
+const Config = mergeComponentMetadata('RangePicker', RangePicker);
+
+storiesOf('Agate', module)
+	.add(
+		'RangePicker',
+		() => (
+			<div style={{padding: '0 20%'}}>
+				<RangePicker
+					disabled={boolean('disabled', Config)}
+					onChange={action('onChange')}
+					min={number('min', Config, 0)}
+					max={number('max', Config, 20)}
+					step={number('step', Config, 1)}
```

Comment:
Please do not define any default value of the boolean, number, select in '@enact/storybook-utils/addons/knobs' if that default value is already defined in our component.

The `step` and the `orientation` are defined in `@agate/internal/Picker`. So here is the way that we apply those values to the `number` and the `select`.

1. Import modules having the defaultProps
2. Append those modules as the parameters of the mergeComponentMetadata function.
3. Omit the default values in number, select, and so on.

Then `Config` will pass the component's default value to boolean, number, and select knobs.

```
import {PickerBase} from '@enact/agate/internal/Picker';

const Config = mergeComponentMetadata('RangePicker', RangePicker, PickerBase);

<RangePicker
	defaultValue={number('defaultValue', Config, 5)}
	disabled={boolean('disabled', Config)}
	key={number('defaultValue', Config, 5)
	onChange={action('onChange')}
	max={number('max', Config, 20)}
	min={number('min', Config, 0)}
	orientation={select2('orientation', ['vertical', 'horizontal'], Config)}
	step={number('step', Config)}
/>
```

Reply:
Oh, you have to export `PickerBase` in internal/Picker first.

Reply:
Thanks for the tip. I was wondering how the defaultValues are brought in this case

Reply:
https://github.com/enactjs/agate/pull/272#discussion_r444638206

As roy mentioned, the defaultValue shouldn't be used in samplers.